expression/builtin_string.go

```diff
@@ -1696,7 +1696,71 @@ type builtinExportSetSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.6/en/string-functions.html#function_export-set
 func (b *builtinExportSetSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("export_set")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	var (
+		bits         string
+		on           string
+		off          string
+		separator    = ","
+		numberOfBits = 64
+	)
+	arg0, err := args[0].ToInt64(b.ctx.GetSessionVars().StmtCtx)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	bits = strconv.FormatInt(arg0, 2)
+	on, err = args[1].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	off, err = args[2].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	switch len(args) {
+	case 4:
+		separator, err = args[3].ToString()
+		if err != nil {
+			return d, errors.Trace(err)
+		}
+	case 5:
```

Comment:
line 1730~1733 is duplicated.
We can remove these line3, and move `case 5 {}`  to the front of `case 4{}`
then add `fallthrough` at the end of `case 5`.